features/contract-state/GeneralInfo.tsx

```diff
@@ -40,11 +41,16 @@ const GeneralInfo = () => {
     priceIdentifier: priceId,
     collateralRequirement: collReq,
     minSponsorTokens,
-    withdrawalLiveness,
   } = empState;
   const { symbol: tokenSymbol } = Token.useContainer();
+
+  const [sponsorCount, setSponsorCount] = useState<string | null>(null);
   const defaultMissingDataDisplay = "N/A";
 
+  useEffect(() => {
```

Comment:
`activeSponsors` is already updated within an `useEffect` hook in the EMPSponsors container, so why not just add `activeSponsors !== null` to the big `if` statement, and then just create a variable `const sponsorCount = Object.keys(activeSponsors).length`? I don't think you need to create the state variable as you did

Reply:
yup, that's a cleaner way of doing it. I didn't know I could access the `activeSponsors` like that from the EMPSponsors container. I have refactored, PTAL.